### ISSUE TITLE
OpenAPI plugin - JSON content type is not actually JSON

### ISSUE THREAD

**Issue by adamwallred** [outofcoffee/imposter#24]
Hello!

I have a swagger spec that, in-part, looks like:

```
responses:
        Team:
            description: success
            schema:
                $ref: '#/definitions/Team'

            examples:
                application/json:
                    id: 10
                    name: Engineering
```
The JSON schema that is produced looks like:

```
"Team": {
      "description": "success",
      "examples": {
        "application/json": {
          "id": 10,
          "name": "Engineering"
        }
      },
      "schema": {
        "$ref": "#/definitions/Team"
      }
    },
```

If I hit the imposter API endpoint configured as such:

```
{
    "plugin": "com.gatehill.imposter.plugin.openapi.OpenApiPluginImpl",
    "specFile": "apiv2.json",
}
```

I'm hoping to see the example returned. What I get is:

`{id=10, name=Engineering}`

which isn't valid JSON.

Looking at curl:

```
~ $ curl -v -H "Accept: application/json" http://127.0.0.1:8443/api/v2/teams/10
*   Trying 127.0.0.1...
* Connected to 127.0.0.1 (127.0.0.1) port 8443 (#0)
> GET /api/v2/teams/10 HTTP/1.1
> Host: 127.0.0.1:8443
> User-Agent: curl/7.49.1
> Accept: application/json
>
< HTTP/1.1 200 OK
< Content-Type: application/json
< Content-Length: 19
<
* Connection #0 to host 127.0.0.1 left intact
```

Imposter claims that this is JSON.

Watching debug output:

```
17:10:28.177 [vert.x-worker-thread-7] DEBUG com.gatehill.imposter.service.ResponseServiceImpl - Using default response behaviour for request: http://127.0.0.1:8443/api/v2/teams/10
17:10:28.179 [vert.x-worker-thread-7] DEBUG com.gatehill.imposter.plugin.openapi.OpenApiPluginImpl - Exact example match found (application/json) from specification
17:10:28.179 [vert.x-worker-thread-7] INFO  com.gatehill.imposter.plugin.openapi.OpenApiPluginImpl - Serving mock example for URI http://127.0.0.1:8443/api/v2/teams/10 and status code 200 (response body 19 bytes)
```

It looks like the example is returned, but not as encoded JSON, but as a string representation of a Map...

**Comment by outofcoffee** [comment]
Hi @adamwallred, thank you for raising this.

This was actually a capability that we've not had to date, but I can see how it would be helpful.

I've raised a PR with rudimentary support for this, supporting JSON and YAML serialised content types in the response: #33

**Comment by outofcoffee** [comment]
Fixed in #33
